vms/platformvm/state/state.go

```diff
@@ -286,10 +286,10 @@ type stateBlk struct {
 type state struct {
 	validatorState
 
-	cfg     *config.Config
-	ctx     *snow.Context
-	metrics metrics.Metrics
-	rewards reward.Calculator
+	validators validators.Manager
```

Comment:
@danlaine  I think the reason we have the full config here is that we used to need more config fields before introducing `config.ExecutionConfig`.
Anyhow now your cleanup makes much sense